tests/command/cursor-tailable-001.phpt

```diff
@@ -67,5 +67,5 @@ object(stdClass)#%d (%d) {
 Waited for 0.%d seconds
 Awaiting results...
 NULL
-Waited for 0.5%d seconds
+Waited for 0.%r(4|5)\d*%r seconds
```

Comment:
MongoDB 3.6 and earlier would typically wait just under 500ms, while 4.0.0-rc5 appeared to wait a bit over. This regex will test that we're waiting somewhere in the range of 400-599 milliseconds, which I think is reasonable to ensure our maxTimeMS is respected.